yarn.lock

```diff
@@ -1407,6 +1407,24 @@
   resolved "https://registry.yarnpkg.com/@fluentui-react-native/design-tokens-windows/-/design-tokens-windows-0.13.0.tgz#1d8610634920bd5decaca669cae8183c0713d2ef"
   integrity sha512-o9ir+tpie+Fm9zO+nCbJLBfzVjUhugRWRsPS8n6wBaujC1qyeUYjhoGg6ejXzKc0nVCkIsFmRRRgC0brfrPIzw==
 
+"@fluentui-react-native/framework@0.7.17":
```

Comment:
I think the dependencies in your Notication's package.json might be out of date. Can you make sure it's depending on the latest version of all the @fluentui-* packages? Their versions are just elsewhere in our repo, also in package.json files